prisma/schema.prisma

```diff
@@ -87,6 +88,13 @@ model Meet {
   guests       String[]
   notes        String?
   tags         Tag[]
-  Venue        Venue   @relation(fields: [venueId], references: [id])
+  Venue        Venue    @relation(fields: [venueId], references: [id])
   venueId      String
 }
+
+model Settings {
+  friendsVisibility Boolean @default(false)
+  profileVisibility Boolean @default(false)
```

Comment:
Please make this the strings, "public", "private" or "only friends".

Same for friendsVisibility.